config/management/src/lib.rs

```diff
@@ -116,22 +122,24 @@ impl Command {
             .get(key)
             .and_then(|c| c.value.u64())
             .map(|c| c.to_string())
-            .unwrap_or_else(|e| "None".into());
+            .unwrap_or_else(|e| format!("{:?}", e));
         writeln!(buffer, "{} - {}", key, value).unwrap();
     }
 
     fn write_waypoint(storage: &dyn Storage, buffer: &mut String, key: &str) {
-        let value = if let Ok(value) = storage.get(key).and_then(|c| c.value.string()) {
-            if value.is_empty() {
-                "empty".into()
-            } else {
-                Waypoint::from_str(&value)
-                    .map(|c| c.to_string())
-                    .unwrap_or_else(|_| "Invalid waypoint".into())
-            }
-        } else {
-            "None".into()
-        };
+        let value = storage
```

Comment:
At some point it would be good to write a test for this logic to make sure the correct values are being displayed in each case. Although I guess this is probably something that we'll only need once we've settled on the output format. A TODO might be good.

Reply:
Once we actually start using the data for something, we will verify correctness. The intent here was to start building a framework for testing a CLI tool.